ProcessLib/Process.cpp

```diff
@@ -50,6 +50,38 @@ Process::Process(
 {
 }
 
+/// Initialize the boundary conditions for single PDE
+void Process::initializeBoundaryConditionPerPDE(
+    const NumLib::LocalToGlobalIndexMap& dof_table, const int process_id)
```

Comment:
Naming issue: A process could consist of multiple PDEs. I think initializeProcessBoundaryCondition or similar would be a better name.

Reply:
Changed the name accordingly.